client/webpack.config.js

```diff
@@ -37,23 +37,33 @@ const { LicenseWebpackPlugin } = require('license-webpack-plugin');
 
 const resourcePath = path.resolve(__dirname + '/resources');
 
+
+const copyPattern = {
+  from: './public'
+};
+
+if (DEV) {
+  copyPattern.transform = applyDevCSP;
+}
+
 module.exports = {
   mode: DEV ? 'development' : (LICENSE_CHECK ? 'none' : 'production'),
   target: 'web',
   entry: {
     bundle: [ './src/index.js' ]
   },
   output: {
-    path: __dirname + '/build',
-    filename: '[name].js',
-    chunkFilename: '[name].[id].js'
+    path: __dirname + '/build'
   },
   resolve: {
-    mainFields: DEV ? [ 'browser', 'dev:module', 'module', 'main' ] : undefined,
+    mainFields: DEV ? [ 'browser', 'dev:module', 'module', 'main' ] : [ 'browser', 'module', 'main' ],
     modules: [
       'node_modules',
       resourcePath
-    ]
+    ],
+    fallback: {
+      util: require.resolve('util/')
```

Comment:
What do we need this for?

Reply:
Similar to https://github.com/camunda/camunda-modeler/pull/3062#discussion_r940995410, `util` is used in the client (e.g. [util/inherits](https://github.com/camunda/camunda-modeler/blob/736fe6cef9d6857b0beef4ee0c0482cba85b298f/client/src/app/tabs/dmn/modeler/features/decision-table-keyboard/DecisionTableKeyboard.js#L11)) and needs to be provided by us in Webpack@5

Reply:
We should rather change to [`inherits-browser`](https://github.com/nikku/inherits-browser) which gets rid of the util import _or_ configure webpack to use the `browser` field over the `main` and `module` fields.